src/lib/genesis_ledger_helper/genesis_ledger_helper.ml

```diff
@@ -571,26 +640,7 @@ module Genesis_proof = struct
         None )
     in
     let filename = filename ~base_hash in
-    match%bind
-      Deferred.List.find_map ~f:(file_exists filename) search_paths
-    with
-    | Some filename ->
-        return (Some filename)
-    | None -> (
-        let s3_path = s3_bucket_prefix ^/ filename in
-        let local_path = Cache_dir.s3_install_path ^/ filename in
-        match%bind
-          Cache_dir.load_from_s3 [ s3_path ] [ local_path ] ~logger
-        with
-        | Ok () ->
-            file_exists filename Cache_dir.s3_install_path
-        | Error e ->
-            [%log info] "Could not download genesis proof file from $uri"
-              ~metadata:
-                [ ("uri", `String s3_path)
-                ; ("error", Error_json.error_to_yojson e)
-                ] ;
-            return None )
+    Deferred.List.find_map ~f:(file_exists filename) search_paths
```

Comment:
Is this required? Can we still log the error and return `None` if the file was not found?